lib/logging/logging.go

```diff
@@ -137,19 +141,38 @@ func (rf *rollingFile) roll() error {
 	return rf.open()
 }
 
-func openLogFile(name string) (f *os.File, err error) {
+func openLogFile(name string, owner string) (f *os.File, err error) {
 	f, err = os.OpenFile(name, os.O_WRONLY|os.O_CREATE|os.O_APPEND, 0644)
+	if err != nil {
+		return
+	}
+	// If owner is defined, change the owner of the log file to this user
+	if owner != "" {
+		ownerUser, err := user.Lookup(owner)
+		if err != nil {
+			return f, err
+		}
+		uid, err := strconv.Atoi(ownerUser.Uid)
+		if err != nil {
+			return f, err
+		}
+		gid, err := strconv.Atoi(ownerUser.Gid)
+		if err != nil {
+			return f, err
+		}
+		os.Chown(name, uid, gid)
```

Comment:
`os.Chown()` doesn't work on Windows. https://golang.org/pkg/os/#Chown
Maybe you need to convert a string SID got from `user.Lookup()` to a real SID format Windows uses by using the function listed [here](https://godoc.org/golang.org/x/sys/windows), and load a DLL and call a Windows function in order to change the file owner.

However, In comparison with the above, I think it will be easier to handle the file owner if the initial file isn't created at installation and created by the process of writing to it.